public/config/configuration.json

```diff
@@ -1,11 +1,12 @@
 {
-  "API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/",
-  "AUTH_URL": "https://dev.bcregistry.ca/cooperatives/auth/",
-  "SIGNIN_URL": "https://dev.bcregistry.ca/cooperatives/auth/signin/bcsc/",
-  "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/",
-  "PAY_API_URL": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/",
-  "KEYCLOAK_CONFIG_URL": "/local-keycloak-config-url/keycloak.json",
-  "ADDRESS_COMPLETE_KEY": "RH99-RN99-GB72-FW86",
-  "LD_CLIENT_ID": "5e5d5af4574f7e09ab53d03b",
-  "VUE_APP_AUTH_ROOT_API": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1"
+    "API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/",
+    "AUTH_URL": "https://dev.bcregistry.ca/cooperatives/auth/",
+    "BUSINESSES_URL": "https://dev.bcregistry.ca/cooperatives/auth/business/",
+    "SIGNIN_URL": "https://dev.bcregistry.ca/cooperatives/auth/signin/bcsc/",
+    "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/",
```

Comment:
We'll need to be careful when merging this one with ENTITY#2960 so as `AUTH_API_URL` is removed

Reply:
@bsnopek-freshworks Let's get yours merged first then!

Reply:
Synced up with master.

Reply:
Updated configuration.json looks OK.